centos8/README.md

```diff
@@ -72,9 +72,9 @@ The timeout to apply when building the image. The default value is set to 1h.
 
 ```shell
 maas $PROFILE boot-resources create \
-    name='centos/8-custom' title='CentOS 8 Custom' \
+    name='custom/centos-8' title='CentOS 8 Custom' \
```

Comment:
it this change really necessary? `centos/your-name` should work just fine

Reply:
Corrected. This was seen during one of the test runs so I had updated it. However, this change is not necessary.